.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "cpp/thirdparty/cnmem"]
 	path = cpp/thirdparty/cnmem
 	url = https://github.com/NVIDIA/cnmem.git
+[submodule "cpp/thirdparty/rmm"]
```

Comment:
Should the above `cnmem` submodule entry be deleted from this file?

Reply:
Yes, will do that.